osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -168,7 +168,7 @@ public BeatmapSetInfo Import(ArchiveReader archiveReader)
 
             // let's only allow one concurrent import at a time for now.
             lock (importLock)
-                connection.RunInTransaction(() => Import(set = importToStorage(archiveReader)));

```

Comment:
what happened to transaction logic?

Reply:
AFAIK SaveChanges method uses transactions under the hood. Also, it is possible to use transactions manually using DbContext.Database.BeginTransaction() method.
https://docs.microsoft.com/en-us/ef/core/saving/transactions

Reply:
Sure, but from what I can see here you removed the transaction and didn’t leave a todo or anything.